api/cases/case-list.js

```diff
@@ -13,67 +23,115 @@ function getOnlineHearing(caseIds, options) {
 
 function rawCasesReducer(cases, columns) {
     return cases.map(caseRow => {
-                return {
-                    case_id: caseRow.id,
-                    case_reference: valueProcessor(sscsCaseListTemplate.case_number.label, caseRow),
-                    case_fields : columns.reduce((row, column) => {
-                        row[column.case_field_id] = valueProcessor(column.value, caseRow);
-                        return row;
-                    }, {})
-                };
-            });
+        return {
+            case_id: caseRow.id,
+            case_jurisdiction: caseRow.jurisdiction,
+            case_type_id: caseRow.case_type_id,
+            case_fields: columns.reduce((row, column) => {
+                row[column.case_field_id] = valueProcessor(column.value, caseRow);
+                return row;
+            }, {})
+        };
+    });
 }
 
-function format(state)
-{
+function format(state) {
     let formattedState = state.split("_").join(" ");
-    return formattedState[0].toUpperCase() +  formattedState.slice(1);
+    return formattedState[0].toUpperCase() + formattedState.slice(1);
 }
 
-//List of cases
-module.exports = (req, res, next) => {
-    const token = req.auth.token;
-    const userId = req.auth.userId;
-    const options = {
-        headers: {
-            'Authorization': `Bearer ${token}`,
-            'ServiceAuthorization': req.headers.ServiceAuthorization
-        }
-    };
 
-    getCases(userId, options)
-        .then(caseData => {
-            let caseIds = caseData.map(caseRow => 'case_id=' + caseRow.id).join("&");
-            let casesData = getOnlineHearing(caseIds, options)
+function hasCOR(caseData) {
+    return caseData.jurisdiction === 'SSCS';
```

Comment:
this should not be hardcode - should be set in idam. Please can we create a user story highlighting that currently jurisdiction and hasCOR is hardcoded and should be the next step in getting multi jurisdiction epic done.

Reply:
please could you add the comment in the code that reference the user story. This is important as it will help outline what is done and what needs to be done. It will also explain to devs why this is currently hardcoded.

Reply:
There is no user story for this

Reply:
then create one? or ask someone to create it?